<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntranceDiag</class>
 <widget class="QDialog" name="EntranceDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>333</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>333</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>333</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>水文水理模式庫</string>
  </property>
  <widget class="QPushButton" name="initCCHE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CCHE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initRESED">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RESED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initSRH">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SRH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initNetSIM">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GNetSIM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initGLMRT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GLMRT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initNTOUHydro">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>水文水理分析系統</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
